### PR TITLE
Change the start time upon change interval

### PR DESCRIPTION
**Pull request by HelenaEksler**
#349

### CODE REVIEW

**Comment by HelenaEksler** [comment]
interval

**Comment by HelenaEksler** [comment]
@amitaibu Ready

File: client/app/scripts/controllers/repo.js [hunk 1]
```diff
@@ -61,15 +61,35 @@ angular.module('clientApp')
       Builds
         .update($scope.build.id, 'ci_build', params)
         .then(function() {
-          $scope.responseStatus = true;
+          if ($scope.ciBuildItemQueueOrInProgress.status == "queue") {
```
Comment (amitaibu) [inline client/app/scripts/controllers/repo.js:66]: this needs some comments

**Comment by HelenaEksler** [comment]
@amitaibu I've added comments.

File: client/app/scripts/controllers/repo.js [hunk 2]
```diff
@@ -61,15 +61,38 @@ angular.module('clientApp')
...
         .then(function() {
-          $scope.responseStatus = true;
+          // If the latest Build item has status 'queue' - update Build item's
+          // start time.
+          if ($scope.ciBuildItemQueueOrInProgress.status == "queue") {
+            var timestamp = parseInt(Date.now() / 1000) + parseInt($scope.build.interval);
+
+            CiBuildItems
+              .update($scope.ciBuildItemQueueOrInProgress.id, {'start_timestamp': timestamp})
+              .then(function() {
```
Comment (amitaibu) [inline client/app/scripts/controllers/repo.js]: I think you can do  `.then(showSuccessMark);`

**Comment by HelenaEksler** [comment]
@amitaibu You are right. Done.

**Pull request merged**